README.md

```diff
@@ -44,6 +44,7 @@ lowercaseOutputName: false
 lowercaseOutputLabelNames: false
 whitelistObjectNames: ["org.apache.cassandra.metrics:*"]
 blacklistObjectNames: ["org.apache.cassandra.metrics:type=ColumnFamily,*"]
+cacheRules: true
```

Comment:
There is no config field by this name, caching is per rule.

Reply:
D'oh! I was mislead by https://github.com/banzaicloud/kafka-operator/pull/465/files#diff-e05b0e4cced2cbda277bdcc359c194d6ae588cffa480c3f1874abb8ad08fa265R602 . Let me re-evaluate